org-code-javabuilder/media/src/main/java/org/code/media/FontHelper.java

```diff
@@ -1,42 +1,20 @@
 package org.code.media;
 
 import java.awt.*;
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.HashMap;
 import java.util.Map;
-import org.code.protocol.InternalErrorKey;
-import org.code.protocol.InternalServerRuntimeError;
+import org.code.protocol.GlobalProtocol;
 
 public class FontHelper {
   private final Map<String, java.awt.Font> fontMap;
 
   public FontHelper() {
-    this.fontMap = new HashMap<>();
-    this.populateFontMap();
+    this.fontMap = GlobalProtocol.getInstance().getCachedResources().getFontMap();
```

Comment:
It seems a bit unexpected to call GlobalProtocol here.  I'm wondering if it would be better to cache the whole FontHelper class and get an instance via GlobalProtocol in Stage (where it looks like we're already getting some items off of GlobalProtocol).

Reply:
the reason I have it here is we need the `FontHelper` to have access to media's `Font` and `FontStyle` classes, which students also need access to. However, students don't have access to `protocol`. I can instead pass in the font map as a constructor parameter so we don't need to reference global protocol here.